setup.py

```diff
@@ -79,7 +79,7 @@
     "opencv-python<=4.6.0.66",
 ]
 _transformers_deps = _pytorch_deps + [
-    "transformers<4.35.0",
+    "transformers<4.40",
     "datasets<=2.14.6",
```

Comment:
```suggestion
    "datasets<2.19",
```
or we could just put a minimum version and leave the ceiling open